src/guides/v2.4/graphql/mutations/add-requisition-list-items-to-cart.md

```diff
@@ -0,0 +1,98 @@
+---
+group: graphql
+title: addRequisitionListItemsToCart mutation
+b2b_only: true
+contributor_name: EY
+---
+The `addRequisitionListItemsToCart` mutation adds requisition list items to cart.
+
+This mutation requires a valid [customer authentication token]({{page.baseurl}}/graphql/mutations/generate-customer-token.html).
+
+{:.bs-callout-info}
+Use the [storeConfig query]({{page.baseurl}}/graphql/queries/store-config.html) with the `btob_website_configuration_requisition_list_active` attribute to determine whether requisition lists are supported.
+
+## Syntax
+
+```graphql
+mutation {
+  addRequisitionListItemsToCart (
+    requisitionListUid: ID
+    requisitionListItemUids: [ID!]
+  ) {
+    AddRequisitionListItemsToCartOutput
+  }
+}
+```
+
+## Example usage
+
+The following example adds products to requisition list.
+
+**Request:**
+
+``` graphql
+mutation {
+  addRequisitionListItemsToCart
+      (
+        requisitionListUid: "1"
+        requisitionListItemUids: ["1","2"]
+      ) {
+      status
+     }
+}
+```
+
+**Response:**
+
+``` json
+{
+  "data": {
+    "addRequisitionListItemsToCart": {
+        "status": "true"
+    }
+  }
+}
+```
+
+## Input attributes
+
+The `addRequisitionListItemsToCart` mutation requires the following input.
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`requisitionListItemUids`| [ID!] | An array of UIDs presenting products to be added to the cart. If no UIDs are specified, all items in the requisition list will be added to the cart
+`requisitionListUid`| ID | The unique ID of the requisition list
```

Comment:
```suggestion
`requisitionListUid`| ID! | The unique ID of the requisition list
```

We just changed the data type for `requisitionListUid` to ID!. I'm not sure which day the merge will happen, but it will be in the code for 2.4.2.